src/meta/duplication/meta_duplication_service.cpp

```diff
@@ -183,10 +183,6 @@ void meta_duplication_service::do_add_duplication(std::shared_ptr<app_state> &ap
                                                   duplication_add_rpc &rpc)
 {
     dup->start();
-    if (rpc.request().freezed) {
-        dup->persist_status();
-        dup->alter_status(duplication_status::DS_PAUSE);
-    }
```

Comment:
What's those code usage? Why would you delete them?

Reply:
In the past, we usually freeze(DS_PAUSE) dup before uploading checkpoint completed. Now we use new status to `auto-control` it, which will keep DS_PREPARE before checkpoint